docs/package.json

```diff
@@ -15,16 +15,17 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.1",
-    "@docusaurus/preset-classic": "2.0.1",
+    "@docusaurus/core": "^2.1.0",
+    "@docusaurus/preset-classic": "^2.1.0",
     "@mdx-js/react": "1.6.22",
     "clsx": "1.2.1",
     "prism-react-renderer": "1.3.5",
     "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react-dom": "17.0.2",
+    "@easyops-cn/docusaurus-search-local": "^0.32.0"
   },
   "devDependencies": {
-    "@docusaurus/types": "2.0.1",
+    "@docusaurus/types": "^2.1.0",
```

Comment:
please pin the versions if possible so that installations are as similar as possible

Reply:
I really do not like that npm does not pin the version when it was pinned before.
Will pay attention to that in the future!